### ISSUE TITLE
Problem when convert cfg model

### ISSUE THREAD

**Issue by franklu323** [qqwweee/keras-yolo3#115]
I met a problem that I cannot run convert.py, can anyone help me with this??
![default](https://user-images.githubusercontent.com/28680833/41443620-2b1632ea-7092-11e8-9aba-00aa64c9df1d.PNG)

**Comment by stefanbo92** [comment]
I think there might be an issue with your yolov3.cfg. Can you check it our show it here?

**Comment by franklu323** [comment]
Yes, you are right, I just double check my cfg file. My cfg file was wrong, I just replace my cfg file and it's working now. Thanks
